moveit_core/robot_trajectory/src/robot_trajectory.cpp

```diff
@@ -630,6 +630,32 @@ void RobotTrajectory::print(std::ostream& out, std::vector<int> variable_indexes
   out.flush();
 }
 
+bool RobotTrajectory::isContinuous(const double threshold)
+{
+  bool is_continuous = true;
+  // Iterate over joint states & check if distance is higher than the given threshold
+  for (std::size_t index = 1; index < waypoints_.size(); ++index)
```

Comment:
We're not protecting this function if the trajectory has 0 or 1 points. I would just return true immediately at the beginning if that's the case.

Reply:
:sweat_smile: